third_party/2and3/pytz/__init__.pyi

```diff
@@ -1,7 +1,7 @@
 # Stubs for pytz (Python 3.5)
 
 import datetime as dt
-from typing import Optional, List, Set, Dict  # NOQA
+from typing import Optional, List, Set, Dict, Union  # NOQA
```

Comment:
The `# NOQA` comment is probably unneeded. We don't have this anywhere else in typeshed.